manifests/09-prometheus-rules.yaml

```diff
@@ -17,3 +17,7 @@ spec:
         message: |
           Image Registry Storage configuration has changed in the last 30
           minutes. This change may have caused data loss.
+  - name: image-registry.rules
```

Comment:
I'm not sure you need to create another rule group, the new rule can be added to the existing `ImageRegistryOperator`.

Reply:
I see no problem in merging them, I was trying to separated alerting from recording rules (not sure if that makes sense though).

When it comes to the naming I noticed that best practices state that it should be named as: "level:metric:operations", on this case we are omitting the "level" and naming it as ":metric:operations", is this intentional ?

Reply:
> we are omitting the "level" and naming it as ":metric:operations", is this intentional ?

yes because you're aggregating away all labels. If you were doing `sum by(foo) (apiserver_v1_image_imports_total)` then the resulting metric would be `foo:apiserver_v1_image_imports`